### ISSUE TITLE
Does Not Start with Windows

### ISSUE THREAD

**Issue by bkrucarci** [flameshot-org/flameshot#2422]
### Flameshot Version

Flameshot v11.0.0 (ad1bf28)

### Installation Type

Using the ready-made package from Github Releases. (Just using the installer (.exe) on Github releases page)

### Operating System type and version

Windows 10 Pro 21H1

### Description

In settings FlameShot set to start with windows. After every day of my PC start, FlameShot does not start with Windows start. I have to manually open the app in Windows start menu

### System Information

Windows 10 Pro 21H1
Here is my settings config file : https://www67.zippyshare.com/v/4cpLGk3R/file.html

**Comment by SlimGary** [comment]
I can confirm; fresh windows 11 install, fresh flameshot install: It doesn't start with the computer

**Comment by LuanGregati** [comment]
Yes. It happens for me either. In a previous version, Flameshot was listed in Task Manager startup apps, now is not there anymore. And in Flameshot settings, I activated "Start with system" option, but still not working.

System: Windows 11 Pro 21H2

**Comment by borgmanJeremy** [comment]
I cannot replicate this. I just tested on windows 11 and its working fine.

**Comment by neeythann** [comment]
I had this problem - I was using lightshot before and forgot to uninstall it after installing flameshot. Hope this helps.

Im using Windows 10 Home Build 19042

**Comment by Satevis15** [comment]
I fix this with modify register entry:
Ordinateur\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run
key Flameshot was : C:\Program Files\Flameshot\bin\flameshot.exe
i modify with : "C:\Program Files\Flameshot\bin\flameshot.exe"

And Flameshot run at windows startup now (Windows 11 French)

**Comment by mmahmoudian** [comment]
@Satevis15 So basically you put the path in double-quotes?!

**Comment by Satevis15** [comment]
> @Satevis15 So basically you put the path in double-quotes?!

Yes :)

**Comment by wonghoi** [comment]
I noticed "Launch at Startup" was checked by default but no registry entry was in "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run". I unchecked and re-checked the box and the entry for the string named "Flameshot" appeared, without the double quotes.

**Comment by rjurga** [comment]
I can confirm the above: unchecking and checking the option creates the entry, but without the double quotes.

**Comment by borgmanJeremy** [comment]
confirmed. The solution is to have flameshot (if on windows) check that the registry entry is set on launch. The issue is the .msi installer does not create the entry during installation so you need to toggle it to have the registry entry created.

**Comment by mattwelke** [comment]
Reproduced. Fresh Flameshot install from GitHub msi file.

Info:

OS Name	Microsoft Windows 11 Pro
Version	10.0.22000 Build 22000
Launch at startup is enabled:
![image](https://user-images.githubusercontent.com/7719209/169423469-b69b6825-5eaa-41e8-9a06-56310dfc3050.png)

**Comment by HoshyarKarimi** [comment]
I found the solution/workaround

1. From the configuration uncheck **Launch at startup**
2. Quit Flameshot from system tray
3. Now _run Flameshot as administrator_ 
4. Open the configuration and check the **Launch at startup** box

from now on flameshot starts at startup and you can confirm by pressing Ctrl+Shift+Esc, check flameshot status is enabled in Startup tab

**Comment by haskellromero** [comment]
On windows 11 I got it to boot by creating a "string" entry in the windows registry at the location:  "\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Run", name: Flameshot, value information: C:\Program Files\Flameshot\bin\flameshot.exe
![image](https://user-images.githubusercontent.com/22428538/192889942-fdeacd64-44fc-4135-a017-eb0c9ab9e9af.png)
